docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ReplaceMissingValues.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ReplaceMissingValues
+    {
+        public static void Example()
+        {
+            // Creating the ML.Net IHostEnvironment object, needed for the pipeline.
```

Comment:
>// Creating the ML.Net IHostEnvironment object, needed for the pipeline. [](start = 12, length = 72)

Following is the standard comments for the `MLContext` in the samples.
`// Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
// as well as the source of randomness.`

Please see other samples in the folder.